src/main/target/common_pre.h

```diff
@@ -370,6 +370,8 @@
 #define USE_GPS_NMEA
 #define USE_GPS_UBLOX
 #define USE_GPS_RESCUE
+#define USE_GPS_LAP_TIMER
+#define USE_CMS_GPS_LAP_TIMER_MENU
```

Comment:
These should be optional or custom defines in build API.
```suggestion
#if defined (USE_GPS_LAP_TIMER)
#define USE_CMS_GPS_LAP_TIMER_MENU
#endif
```
With this no changes needed in common_pre.h anymore.

Reply:
With BFC 10.9 / 10.10 user can add this using Custom Defines
But we also can ask @blckmn to add it to Other Options.

Reply:
![image](https://user-images.githubusercontent.com/8344830/236288826-397f33cb-3757-4218-abcc-19b8197bcb4d.png)